.github/workflows/test.yaml

```diff
@@ -25,6 +25,13 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: golangci-lint
-      uses: golangci/golangci-lint-action@v3
-      with: # TODO: remove this when the deprecated function will be removed (issue #85)
-        args: --exclude SA1019 --timeout=5m
+      run: golangci/golangci-lint-action@v3
```

Comment:
In a follow up iteration we can turn it into make lint, so we can avoid to keep the linter version in sync